ormconfig-address.json

```diff
@@ -5,13 +5,13 @@
   "logging": true,
   "database": "address-dev.sqlite",
   "entities": [
-     "packages/neuron-wallet/src/database/address/entities/**/*.ts"
+     "packages/neuron-wallet/dist/database/address/entities/**/*.js"
```

Comment:
@classicalliu Could you take a look and test if with these changes our db cli tasks still work? Thanks!

Reply:
It works, but in my mind, we should run `yarn build` before `db:chain` and `db:address`